source/_integrations/lektrico.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: Lektrico Charging Station
+description: Instructions on how to integrate a Lektrico Chargering Station with Home Assistant.
+ha_category:
+  - Sensor
+ha_release: "2024.10"
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@lektrico'
+ha_domain: lektrico
+ha_zeroconf: true
+ha_platforms:
+  - sensor
+---
+
+The **Lektrico Charging Station** integration integrates your [Lektrico Charging Station](https://lektri.co) into your Home Assistant and allows you to monitor it.
+
+The Lektrico Charging Station device will be added as a sensor in Home Assistant.
+
+{% include integrations/config_flow.md %}
+
```

Comment:
Please add a section called ## Sensors, listing the names of the sensors with a description what they do, what unit is used.

Reply:
I added the section  ## Sensors, thank you.

Reply:
Nice!